generators/server/templates/_pom.xml

```diff
@@ -941,9 +963,6 @@
                     Enable the line below to have remote debugging of your application on port 5005
                     <jvmArguments>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</jvmArguments>
                     -->
-                    <arguments>
-                        <argument>--spring.profiles.active=dev</argument>
```

Comment:
Hm, I haven't seen that `-Drun.profiles` option before... So I didn't try that, indeed. Good catch.

But according to [this](http://docs.spring.io/spring-boot/docs/current/maven-plugin/examples/run-profiles.html) it might not be exactly equivalent to the `<arguments>` block we were talking about here? Instead maybe there should be a `<profiles>` block?